contracts/src/v0.8/llo-feeds/FeeManagerNoNative.sol

```diff
@@ -0,0 +1,482 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.16;
```

Comment:
@ad0ll we can update this to 0.8.19 as that is the latest version we're considering stable